app/src/pages/counterPage/components/CounterView/index.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import styles from './styles.module.scss';
+
+const CounterView = ({countValue, handleReset, handleIncrement, handleDecrement, isEven}) => {
+    return (
+        <div className={`${styles.wrapper}, ${(countValue % 2 === 0 ? styles.red: styles.green)}`}>
```

Comment:
you have isEven field in your state, why did you calculate it againg? Use `isEven` instead of `countValue % 2 === 0`